base/applications/games/winmine/lang/ar-AR.rc

```diff
@@ -0,0 +1,83 @@
+/*
+ * PROJECT:    WineMine
+ * LICENSE:    GPL - See COPYING in the top level directory
```

Comment:
license should most likely be LGPL. GPL is outdated in most of our files, and was used as license just in historic files. I do assume you copy pasted this from some existing file, right?